spec/Spec/BlsHelpers.cry

```diff
@@ -163,8 +163,8 @@ decompress_G1 z =
 /**
  * Deserialize the G1 bytes into a G1 point
  */
-bytes48_to_g1_point : Bytes48 -> G1Point
-bytes48_to_g1_point bytes = { xCoord = x, yCoord = y} where
+bytes48_to_G1 : Bytes48 -> G1Point
+bytes48_to_G1 bytes = { xCoord = x, yCoord = y} where
```

Comment:
question: this commit says the name change is to match the spec. Is there a link to add here?

Reply:
Good catch! I ended up adding it here a couple of days ago: 66a660ce0d07d7d557ed1a93e94d1ab34c434f2a